static/src/javascripts/projects/common/modules/commercial/adblock-banners.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+import config from 'lib/config';
+import { inlineSvg } from 'common/views/svgs';
+
+// this is used in all banners
+const cursor = inlineSvg('cursor');
+
+const banners: Array<{
+    template: string,
+    defaults: {},
+    variants: Array<{}>,
+    editions: {},
```

Comment:
Move the type into it's own type?
And maybe use `Object` instead of `{}`? I don't think it even matters though